.circleci/config.yml

```diff
@@ -390,8 +367,10 @@ jobs:
                 -v /tmp/ds001771/work:/work \
                 --user $(id -u):$(id -g) \
                 nipreps/dmriprep:latest /data /out participant -vv \
-                --fs-subjects-dir /data/derivatives/freesurfer-6.0.1 --sloppy \
-                --notrack --skip-bids-validation -w /work --omp-nthreads 2 --nprocs 2
+                -w /work --omp-nthreads 2 --nprocs 2 \
+                --notrack --skip-bids-validation --sloppy \
+                --fs-subjects-dir /data/derivatives/freesurfer-6.0.1 \
```

Comment:
I'm a little bit confused about specifying both `--fs-subjects-dir` and `--anat-derivatives`. Is the idea to have only one of these arguments exist in the future?

Reply:
It's not, in principle. These two flags give the flexibility to run FS in isolation and then connect it to further anatomical processing, again in isolation. Assuming that both go together brings us back to the original situation where we need to have freesurfer, smriprep and dmriprep folders under the output folder, if one wants to reuse cached results.